### PR TITLE
Tests: Copied publish_in_test to AE

### PR DESCRIPTION
**Pull request by kalisp**
Previously only in webpublisher addon, which caused unwanted dependency. Now it duplicates code in AE (and also PS), but its just simple code for automatic testing.

2 lib functions could be potentially moved to ayon-pipeline-tests, but that publish_in_test in addon should stay.

### CODE REVIEW

File: client/ayon_aftereffects/api/lib.py [hunk 1]
```diff
@@ -162,3 +163,47 @@ def set_settings(frames, resolution, comp_ids=None, print_msg=True):
...
+    Args:
+        log (Logger)
+        close_plugin_name (Optional[str]): Name of plugin with responsibility
+            to close application.
+    """
+
+    # Error exit as soon as any error occurs.
+    error_format = "Failed {plugin.__name__}: {error} -- {error.traceback}"
+    log.info("here")
+    print("here")
```
Comment (BigRoy) [inline client/ayon_aftereffects/api/lib.py:190]: Here here. :)

Pretty sure this was some debugging going on.

File: client/ayon_aftereffects/api/launch_logic.py [hunk 2]
```diff
@@ -47,16 +47,15 @@ def main(*subprocess_args):
     launcher = ProcessLauncher(subprocess_args)
     launcher.start()
 
-    if os.environ.get("HEADLESS_PUBLISH"):
+    if os.environ.get("AYON_IN_TESTS"):
```
Comment (iLLiCiTiT) [inline client/ayon_aftereffects/api/launch_logic.py:50]: I think we should use `is_in_tests` from `ayon_core.lib` so we don't have to tackle the env variable at all places.

Comment (kalisp) [reply client/ayon_aftereffects/api/launch_logic.py]: Replaced.

File: client/ayon_aftereffects/api/lib.py [hunk 3]
```diff
@@ -3,6 +3,7 @@
 import json
 import contextlib
 import logging
+import pyblish
```
Comment (iLLiCiTiT) [inline client/ayon_aftereffects/api/lib.py:6]:
```suggestion
import pyblish.api
```

Comment (iLLiCiTiT) [inline client/ayon_aftereffects/api/lib.py:6]: Not sure if this works all the time, but works in console from tray so I guess it might be ok.

**Review by iLLiCiTiT — APPROVED** [review]

**Pull request merged**